packages/shared_preferences/shared_preferences_tool/lib/src/shared_preferences_state_provider.dart

```diff
@@ -236,7 +236,7 @@ class _SharedPreferencesStateProviderState
   void initState() {
     super.initState();
     final EvalOnDartLibrary eval = EvalOnDartLibrary(
-      'package:shared_preferences/shared_preferences.dart',
+      'package:shared_preferences/src/shared_preferences_async.dart',
```

Comment:
we need to eval the exact library, evaluating a exports file throws a not found error.